sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -498,4 +500,73 @@ public EncryptedBlobClientBuilder serviceVersion(BlobServiceVersion version) {
         this.version = version;
         return this;
     }
+
+    /**
+     * Configures the builder based on the passed {@link BlobClient}. This will set the {@link HttpPipeline},
+     * {@link URL} and {@link BlobServiceVersion} that are used to interact with the service.
+     *
+     * <p>Use this method after setting the key in {@link #key(AsyncKeyEncryptionKey, String) key} and keyResolver in
+     * {@link #keyResolver(AsyncKeyEncryptionKeyResolver)}.</p>
+     *
+     * <p>If {@code pipeline} is set, all other settings are ignored, aside from {@link #endpoint(String) endpoint} and
+     * {@link #serviceVersion(BlobServiceVersion) serviceVersion}.</p>
+     *
+     * @param blobClient BlobClient used to configure the builder.
+     * @return the updated EncryptedBlobClientBuilder object
+     * @throws NullPointerException If {@code containerClient} is {@code null}.
+     */
+    public EncryptedBlobClientBuilder blobClient(BlobClient blobClient) {
```

Comment:
I thought you were going to call these pipeline() so it's clear they copy the pipeline?